### ISSUE TITLE
Is it possible not to use any Lyra tokenizer ?

### ISSUE THREAD

**Issue by bacloud22** [LyraSearch/lyra#114]
Hi,
Thanks for providing Lyra !

So i've already relied on a tokenizer with my algorithm, is it possible to pass plain arrays of words ? or I need to re-join the words and go again through the internal tokenizer of Lyra ?

**Comment by micheleriva** [comment]
Hi there,
yes, this is gonna be part of an upcoming release. I'll notify you as soon as we have it!

**Comment by bacloud22** [comment]
If you need to add a persistence (of the model, or index or anything) I can make a **proto** model to help performance. I've been playing with lately. But then it's gonna be an additional dependency (maybe optional).

**Comment by micheleriva** [comment]
Hi @bacloud22 ,
custom tokenizer is available in [v0.2.1](https://github.com/LyraSearch/lyra/releases/tag/0.2.1). I am working on the docs right now, but here's how it works:

```js
import { create } from '@lyrasearch/lyra'

create({
  schema: {},
  tokenizer: {
    // example
    tokenizerFn: text => text.split(","),
  }
})
```

About data persistence, we already have a plugin for that: https://github.com/LyraSearch/plugin-data-persistence

We already tested protobuffers, but found out that dpack might perform better in our case. Feel free to open a PR anyway!

**Comment by bacloud22** [comment]
Cool stuff ! thanks a lot for the quick feature.
